ee/clickhouse/sql/events.py

```diff
@@ -0,0 +1,108 @@
+DROP_EVENTS_TABLE_SQL = """
+DROP TABLE events
+"""
+
+DROP_EVENTS_WITH_ARRAY_PROPS_TABLE_SQL = """
+DROP TABLE events_with_array_props_view
+"""
+
+DROP_MAT_EVENTS_WITH_ARRAY_PROPS_TABLE_SQL = """
+DROP TABLE events_with_array_props_mv
+"""
+
+DROP_MAT_EVENTS_PROP_TABLE_SQL = """
+DROP TABLE events_properties_view
+"""
+
+EVENTS_TABLE_SQL = """
+CREATE TABLE events
+(
+    id UUID,
+    event VARCHAR,
+    properties VARCHAR,
+    timestamp DateTime64(6, 'UTC'),
+    team_id Int32,
+    distinct_id VARCHAR,
+    elements_hash VARCHAR,
+    created_at DateTime
+) ENGINE = MergeTree()
+PARTITION BY toYYYYMM(timestamp)
+ORDER BY (id, timestamp, intHash32(team_id))
+SAMPLE BY intHash32(team_id)
+"""
+
+INSERT_EVENT_SQL = """
+INSERT INTO events SELECT generateUUIDv4(), %(event)s, %(properties)s, %(timestamp)s, %(team_id)s, %(distinct_id)s, %(element_hash)s, now()
+"""
+
+GET_EVENTS_SQL = """
+SELECT * FROM events_with_array_props_view
+"""
+
+EVENTS_WITH_PROPS_TABLE_SQL = """
+CREATE TABLE events_with_array_props_view
+(
+    id UUID,
+    event VARCHAR,
+    properties VARCHAR,
+    timestamp DateTime64(6, 'UTC'),
+    team_id Int32,
+    distinct_id VARCHAR,
+    elements_hash VARCHAR,
+    created_at DateTime,
+    array_property_keys Array(VARCHAR),
+    array_property_values Array(VARCHAR)
+) ENGINE = MergeTree()
+PARTITION BY toYYYYMM(timestamp)
+ORDER BY (team_id, created_at, id)
+SAMPLE BY id
+"""
```

Comment:
I'm going to update this one as well with new configs. I'll get this done as soon as I have a good story for how to connect to clickhouse production env.